tests/unit/test_github_session.py

```diff
@@ -146,15 +151,43 @@ def test_token_auth_disables_basic_auth(self):
         s = self.build_session()
         s.auth = ('foo', 'bar')
         s.token_auth('token goes here')
-        assert s.auth is None
+        assert s.auth != ('foo', 'bar')
```

Comment:
How about another assertion, e.g.,

```py
assert isinstance(s.auth, github3.session.TokenAuth)
```

Reply:
Added, or did you want me to replace the existing assertion?